engine/src/main/java/org/terasology/engine/core/subsystem/lwjgl/LwjglGraphics.java

```diff
@@ -60,6 +60,12 @@ public void initialise(GameEngine gameEngine, Context rootContext) {
         this.engine = gameEngine;
         this.context = rootContext;
         this.config = context.get(Config.class).getRendering();
+        if (config.getWindowHeight()<=0 && config.getWindowWidth()<=0){
+            config.setWindowPosX(379);
+            config.setWindowPosY(235);
+            config.setWindowWidth(1280);
+            config.setWindowHeight(800);
```

Comment:
these values are pretty arbitrary. you only se to set the width and height since position is irrelevant. probably should define a constant for a default resolution.

Reply:
These are the values that are generated when we delete the config file and reopen the game. So i took them as default values.

Reply:
and what generates those? Is there a default config file that gets copied in, or are they generated by some method in the config system? or do we somehow skip this bit entirely when there's no config file, and the defaults end up being something lwjgl picked when we didn't give it any other instructions?